packages/framework-core/src/services/event-store.ts

```diff
@@ -17,90 +17,72 @@ const originOfTime = new Date(0).toISOString() // Unix epoch
 export class EventStore {
   public constructor(readonly config: BoosterConfig) {}
 
+  /**
+   * Will fetch the latest snapshot for an entity by applying a reduction
+   * since the time of creation of the last snapshot, UNLESS `pendingEnvelopes`
```

Comment:
I think we don't have pendingEvenlopes here right?

Reply:
Good catch :-D